### DISCUSSION TITLE
Notification Redirect

### DISCUSSION THREAD

**Discussion by kanger87** [inspireui/support#5179] [Working on Features]
1. Notify Message
2. Order status , Chat etc

Hi, we having difficulty on notification redirect as listed above.

Scenario :
When we tap on the notification inside of notify message with launch url contain (product link example),it do not lead back to the relevant link. Deeplink has been setup

When someone texting vendor , made purchase etc, it will get notification too. But the problem is it won't redirect to the relevant tab will tap on the notification. It only leads to open homepage of the app.

Thank you and looking forward for your guidance.

**Comment by Haniel5** [comment]
Please refer to https://youtu.be/dBXmzUFRy2w
